Framework/Geometry/src/Instrument.cpp

```diff
@@ -755,7 +753,7 @@ void Instrument::getBoundingBox(BoundingBox &assemblyBox) const {
       m_cachedBoundingBox = new BoundingBox();
       ComponentID sourceID = getSource()->getComponentID();
       // Loop over the children and define a box large enough for all of them
-      for (auto component : m_children) {
+      for (const auto component : m_children) {
```

Comment:
Can this be a reference?
```suggestion
      for (const auto &component : m_children) {
```